cli.md

```diff
@@ -338,25 +338,41 @@ Usage: guardian workflows remove-guardian-comments [options]
 
 ### Options
 
-Also supports [GitHub Options](#github-options) and [Retry Options](#retry-options).
+Supports [Platform Options](#platform-options), [GitHub Options](#github-options) and [Retry Options](#retry-options).
 
-* **-pull-request-number="100"** - The GitHub pull request number to remove plan
-  comments from. The default value is "0".
-* **-for-command="plan,apply"** - The Guardian command to remove comments for. Valid values are ["apply", "plan"]
+## Policy fetch-data
 
-## Workflows validate-permissions
+Fetch data used for policy evaluation.
 
-Validate a list of allowed permissions for the actor running the current GitHub workflow.
+Usage: guardian policy fetch-data [options]
 
-Usage: guardian workflows validate-permissions [options]
+### Prerequisites
+
+* Required GitHub [permissions](#guardian-cli).
+
+* **members: read** is required for `--include-teams` flag.
+
+### Options
+
+Supports [Platform Options](#platform-options), [GitHub Options](#github-options) and [Retry Options](#retry-options).
+
+* **-output-dir="example/dir"** - Write the policy data JSON file to a target local directory.
+* **--include-teams** - If true, includes team data in payload. Requires 'members: read' token permissions."
+
+
+## Policy enforce
+
+Enforce a set of Guardian policies
+
+Usage: guardian policy enforce [options]
 
 ### Prerequisites
 
-* Read permission to the target GitHub repository `content`.
+* Required GitHub [permissions](#guardian-cli).
```

Comment:
Are there permissions that should be listed here? Or are we linking to a different spot, we should be consistent because this differs in the section right about this, where it lists the permissions

Reply:
Added permissions by supported enforcement types.